docs/overview/sapi.rst

```diff
@@ -89,22 +258,29 @@ runs these same configuration steps):
   :start-after: cli-example-config-start-marker
   :end-before: cli-example-config-end-marker
 
-2. Enter the SAPI information (e.g. your API token) found as described in the section
-   above. To get started, create a minimum configuration by accepting the command's
-   defaults (pressing Enter) for all prompts except the API token (Leap users) or
-   API token and endpoint (on-premises users). You can in the future update the
-   file if needed.
+2. Enter the SAPI information (e.g. your API token) found as described in the
+   :ref:`sapi_access_interacting` section above. To get started, create a minimum
+   configuration by accepting the command's defaults (pressing Enter) for all
+   prompts except the API token\ [#]_\ . You can in the future update the file
+   if needed.
+
+.. [#] Users of on-premises systems should also enter the SAPI endpoint. Users
+   interested in using solvers hosted outside their default region can also
+   configure the SAPI endpoint (see the :ref:`sapi_intro_multiregion` section for
+   the recommended way of configuring such access).
 
 Alternatively, you can create and edit a
 :doc:`D-Wave Cloud Client configuration file </docs_cloud/sdk_index>`
 manually.
 
-You can always set or override the solver, API token, and URL directly in your code
-or as local environment variables. For more information, see the examples in this
-document or :doc:`D-Wave Cloud Client </docs_cloud/sdk_index>`.
+You can always set or override your cofigurations in your code or with environment
+variables. For more information, see the examples in this document or
+:doc:`D-Wave Cloud Client </docs_cloud/sdk_index>`.
+
+.. _sapi_access_config_verify:
 
-Verifying Your Configuration
-----------------------------
+Verify Your Configuration
+=========================
```

Comment:
Instead of 
```
dwave ping -s '{"qpu": true}'
```
maybe it's nicer to write:
```
dwave ping --client qpu
```

It's not ideal, and we might add `--qpu` switch (et al).

Reply:
Unrelated, I think using actual endpoint/solver in the example output would be much clearer than using fake ones.

Reply:
Updated